### PR TITLE
Display the Default Search Form

### PR DESCRIPTION
**Pull request by willwill96**
Starts: https://github.com/connexta/revelio/issues/269
Also fixes a bug where workspaces aren't showing the owner

Screenshot: 
image

### CODE REVIEW

File: src/main/webapp/components/index-cards/index-cards.js [hunk 1]
```diff
@@ -87,6 +95,7 @@ export const AddCardItem = props => {
 
 export const IndexCardItem = props => {
   const {
+    starred,
```
Comment (samuelechu) [inline src/main/webapp/components/index-cards/index-cards.js]: :pencil2: `starred --> default/isDefault`?

File: src/main/webapp/components/search-forms/container.tsx [hunk 2]
```diff
@@ -153,7 +162,11 @@ export default () => {
   }
 
   const forms = getIn(data, ['metacardsByTag', 'attributes'], [])
-
+  const userDefaultForm = getIn(
+    data,
+    ['user', 'preferences', 'querySettings', 'template', 'id'],
+    null
```
Comment (samuelechu) [inline src/main/webapp/components/search-forms/container.tsx]: I think the value of this field will always be null or initialized https://github.com/graphql/graphql-js/issues/1298#issuecomment-377365916

File: src/main/webapp/components/index-cards/index-cards.js [hunk 3]
```diff
@@ -101,6 +110,13 @@ export const IndexCardItem = props => {
         <CardHeader
           title={title}
           subheader={subHeader || moment(modified).fromNow()}
+          action={
+            starred && (
+              <IconButton disabled>
+                <StarIcon color="primary" />
+              </IconButton>
+            )
+          }
```
Comment (willwill96) [inline src/main/webapp/components/index-cards/index-cards.js]: Make the prop action and not starred

File: src/main/webapp/components/index-cards/index-cards.js [hunk 4]
```diff
@@ -45,6 +45,14 @@ export const ShareAction = props => {
...
 }
 
+export const MakeDefaultAction = props => {
+  return (
+    <IconButton onClick={onClick(props.onClick)}>
+      <StarIcon />
+    </IconButton>
+  )
+}
+
```
Comment (willwill96) [inline src/main/webapp/components/index-cards/index-cards.js]: delete

**Pull request merged**